### PR TITLE
Ea/ruby upgrade

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

**Review by andyzito — APPROVED** [review]

File: Gemfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ gem 'rails_drivers'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', group: :doc
 
-gem 'nokogiri', '>= 1.12.5' # >= 1.11.0.rc4 due to CVE-2020-26247
+gem 'nokogiri', '>= 1.13.4' # >= 1.11.0.rc4 due to CVE-2020-26247
```
Comment (awisema) [inline Gemfile]: Should we move to 1.13.6 for [CVE-2022-29181](https://nvd.nist.gov/vuln/detail/CVE-2022-29181) as long as we are reving it? Hopefully, we can remove it shortly.

File: app/controllers/concerns/health_careplan.rb [hunk 2]
```diff
@@ -32,9 +33,25 @@ def careplan_pdf_coversheet
...
+        margin: {
+          top: '.5in',
+          bottom: '.5in',
+          left: '.4in',
+          right: '.4in',
+        },
+        wait_until: 'networkidle0',
+        print_background: true,
+      }
+      coversheet = "<html><head><meta charset='UTF-8' /></head><body>#{coversheet}</body></html>"
```
Comment (awisema) [inline app/controllers/concerns/health_careplan.rb]:
```suggestion
      coversheet = wrap_in_html(coversheet)
```

and add
```
private def wrap_in_html(content)
  "<html><head><meta charset='UTF-8' /></head><body>#{content}</body></html>"
end
```

So we can have a single truth for this?

File: app/controllers/concerns/health_careplan.rb [hunk 3]
```diff
@@ -44,13 +61,35 @@ def careplan_pdf_full
...
+        margin: {
+          top: '1.5in',
+          bottom: '1.5in',
+          left: '.4in',
+          right: '.4in',
+        },
+        wait_until: 'networkidle0',
+        print_background: true,
+      }
+      pctp = "<html><head><meta charset='UTF-8' /></head><body>#{pctp}</body></html>"
```
Comment (awisema) [inline app/controllers/concerns/health_careplan.rb]:
```suggestion
      pctp = wrap_in_html(pctp)
```

File: app/controllers/concerns/health_careplan.rb [hunk 4]
```diff
@@ -60,13 +99,35 @@ def careplan_pdf_pctp
...
```
Comment (awisema) [inline app/controllers/concerns/health_careplan.rb]:
```suggestion
      pctp = wrap_in_html(pctp)
```

**Review by awisema — CHANGES_REQUESTED** [review]

**Review by awisema — APPROVED** [review]

**Pull request merged**